### DISCUSSION TITLE
Make Spinner Max Easier

### DISCUSSION THREAD

**Discussion by WalmartWhiteCat** [ppy/osu#25431] [Ideas]
**Opening statement(s)** - Okay, so I've been struggling with whether or not I should post this for a little bit, but I personally feel I have to say something. Now, this could simply be a strictly me thing, but as someone who spins an average of 380 - 400 rpm, I have rarely hit max score on a spinner. Maybe I just don't spin soon enough or something, I have no clue.

**Main idea** - If my issue is not a personal thing, I suggest making it easier to hit max on spinners. It is incredibly - at least to me - frustrating/upsetting to not hit something that is said to be "easy" or "simple" for how fast I spin, as well as having that occurence of not hitting max be constant.

If making it simply easier to hit max score on a spinner is a no go, then maybe make it so the difficulty of doing so is not dependant on OD. I know spinners are usually tied to OD, but in this case I don't think that's the best plan. Also, please let me know if this is already how the max score works, or if it does to at least some degree, because I have no idea.

**End of main idea** - So, instead of it being dependant on OD, maybe make it dependant on how much score can be gotten in the map, which is 1m. So basically make the score max out depending on how much score has already been gotten and how much will be gotten.

**Closing thoughts** - I know I'm terrible at explaining things like this, but I really hope my idea has gotten through in my words, since I really want to convey how annoying it is to not hit max when I know I should easily be able to. Of course, it's not a perfect system, but I know I won't be the only one to complain about this if it isn't strictly a personal experience with spinners. I hope I haven't forgotten anything. Thanks for reading.

**Comment by peppy** [comment]
That sounds weird, as you shouldn't need 380-400rpm to hit max. Can you give an example replay / beatmap?

**Reply by WalmartWhiteCat** [reply]
There's no real replay example I can give, as I would have to record an entire map and I suck at editing software so you would have to watch the entirte thing, and I really don't want to subject you to that. But I can give you a beatmap where the difficulty of getting max score is at the very least incredibly challenging for someone of my level.

A very good example would be a map by the name of Little Cry Baby, specifically the Extra diff: 
https://osu.ppy.sh/beatmapsets/1098258#osu/2374611

In this diff, there is a spinner at the 1:45 timestamp that plays right in the middle of a decently intense section of the map that has little to no build up at all to the actual spinner. This makes it a bit unnecessarily challenging to reach any sort of spin speed sufficient to max out this particular spinner. Even with Auto it only just gets 2 ticks on the max. **Auto**. Additonally, there is another spinner at 4:24 that is exactly the same, with it being rather sudden and in the middle of one of the harder parts of the map.

There are more examples that I could give if I were to be just a little more thorough, but I'm sure with just this map you can see my point. With the lack of buildup spinners like this have, it is very difficult for a human at the skill level of this map to actually get max, especially with how instantly the notes start back up after the spinners, making it much more likely for players going for max to miss right after; another decently large issue I see with how max currently works.

**Reply by peppy** [reply]
That beatmap is OD 9, which means it requires very high precision and spin speed.

For the spinner in question, the raw speed expectation is 420, but after various roundings occur, it works out to 383.8.

To confirm, here's auto maxing it out at 383 SPM:

https://github.com/ppy/osu/assets/191335/3416ea34-8633-4894-9d1d-11017d59f118

I think this seems okay? Arguably, there could be other beatmaps that (due to rounding) require closer to 420, but the one you specify looks very obtainable.

**Reply by WalmartWhiteCat** [reply]
Oh, okay then it probably is just a me thing. Sorry for possibly wasting your time. But I'm actually glad to learn this is a me thing, I would hate it if this addition really was broken. I like it.
